.github/actions/abort-release-candidate-v1/action.yml

```diff
@@ -12,6 +12,9 @@ inputs:
     description: 'The docs repository where the issue for the release candidate is located'
     default: ''
     required: false
+  version:
```

Comment:
I agree with @Zidious 's original comment - I don't think this is the right way to solve the problem. If the goal is support for releases that have an unmerged RC PR, why not just look for PRs with `label: release` and infer the version from that?

Reply:
(could maybe generalize `is-release-in-progress` to `get-release-in-progress` to be able to reuse it)